yarn.lock

```diff
@@ -3756,14 +3749,21 @@
     "@types/react" "*"
     "@types/react-native" "*"
 
-"@types/react-native@*", "@types/react-native@^0.60.22":
+"@types/react-native@*":
```

Comment:
We should be able to delete this entry and yarn install again so that there's only one version of the types loaded.